### DISCUSSION TITLE
🆕 Want to participate in Codespaces customer research?

### DISCUSSION THREAD

**Discussion by jnstuart** [community/community#12621] [Codespaces]
The GitHub team is looking to get feedback from customers on new designs for Codespaces repository permissions. We are looking to chat with current Codespaces customers - **no specific experience necessary!** 

We are happy to provide a **$125 gratuity (gift card or donation)** as a thank you for your time.
If you are interested participating in a virtual one-hour customer research session, please complete the form below. We will reach out to you to schedule a session if we have available time slots.

[Sign up here!](https://github.surveymonkey.com/r/6PM6WMB)

**Comment by 3kh0** [comment]
Me who does not have codespaces: 😞

**Comment by Alemenew** [comment]
i want to participate on python development.

**Comment by Chnistolyonnais** [comment]
I did not understand anything they taught me

**Comment by volksec** [comment]
I want to participate *-*
